lib/locales/pl.ts

```diff
@@ -40,8 +40,8 @@ export const locale: Locale = {
 
     easter_time: {
       season: 'Wielkanoc',
-      weekday: '$t(weekdays:{{dow}}, capitalize) {{week}}. tygodnia wielkanocnego',
-      sunday: '{{week}}. Niedziela Wielkanocna',
+      weekday: '$t(weekdays:{{dow}}, capitalize) {{week}} tygodnia wielkanocy',
```

Comment:
Isn’t this a typo? Should it not be `Wielkanocy`? 🤔

Reply:
Correct, it's a typo. But actually the full name, as printed in the lectionary, would be `tygodnia okresu Wielkanocy`.

Reply:
Therefore, should it be rather `tygodnia okresu Wielkanocy`? :thinking: It seems like you prefer `tygodnia Wielkanocy` for some reason. AFAIK, the weekday celebration names are not printed in full in the Roman Missal (they are simply titled by weekday name, e.g. `Monday` unless they are some special days). :thinking:

Reply:
>  AFAIK, the weekday celebration names are not printed in full in the Roman Missal (they are simply titled by weekday name, e.g. `Monday` unless they are some special days). 🤔

In polish missal they are printed in full, i.e. `Sobota drugiego tygodnia Wielkanocnego`.

Anyway, let's keep the change I made as a best compromise.